plot_dataset.py

```diff
@@ -0,0 +1,283 @@
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+from typing import Sequence
+from functools import reduce
+from itertools import product
+
+
+vox_views = [['sag', 'vox', 50], ['ax', 'vox', 50], ['cor', 'vox', 50]]
+mm_views = [['sag', 'mm', 0], ['ax', 'mm', 0], ['cor', 'mm', 0]]
+
+
+class PlotDataset:
+    """Draw an interactive plot of a few subjects from a torchio dataset.
+    Scrolling on the different images enable to navigate sections.
+
+    Args:
+        dataset: a torchio.ImagesDataset
+        views: None or a sequence of views, each view is given as (view_type, coordinate_system, position),
+            view_type is one among "sag", "ax" and "cor" which corresponds to sagittal, axial and coronal slices;
+            coordinate_system is one among "vox" or "mm" and is responsible for placing the slice using position
+            in term of voxel number or number of millimeters;
+            position is either a number between 0 and 1 if coordinate_system is "vox" or an integer otherwise.
+            If no value is provided, the default views are used:
+            [['sag', 'vox', 50], ['ax', 'vox', 50], ['cor', 'vox', 50]] or
+            [['sag', 'mm', 0], ['ax', 'mm', 0], ['cor', 'mm', 0]] if coordinate_system is 'vox' or 'mm'.
+        coordinate_system: a string that define the views to use in case no value is given.
+        view_org: None or a sequence of length 2, responsible for the organisation of views in subplots.
```

Comment:
why not a default value :  (1, len(view) )

Reply:
This is the idea but we cannot access the `views` attribute at this point.